include/spdk_internal/bdev.h

```diff
@@ -271,6 +271,13 @@ struct spdk_bdev_io {
 		struct {
 			enum spdk_bdev_reset_type type;
 		} reset;
+		struct {
+			struct spdk_nvme_cmd cmd;
+			void *buf;
+			size_t nbytes;
+			uint32_t p_lcore;
+			struct spdk_nvme_cpl cpl;
```

Comment:
Copying the completion is an implementation detail of the blockdev_nvme module, so put a copy of the spdk_nvme_cpl in struct nvme_bdev_io instead of here.